Source/Filter/Filter.cpp

```diff
@@ -168,7 +168,7 @@ Filter::ApplyStencil (amrex::MultiFab& dstmf, const amrex::MultiFab& srcmf, int
 {
     WARPX_PROFILE("Filter::ApplyStencil(MultiFab)");
     ncomp = std::min(ncomp, srcmf.nComp());
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
 #pragma omp parallel
```

Comment:
Missing `if (amrex::Gpu::notInLaunchRegion())`?

Reply:
No.  But we should add a comment.  This is in the else part of `#ifdef AMREX_USE_GPU`.

Reply:
```suggestion
// never run on GPU since in the else branch of AMREX_USE_GPU
#pragma omp parallel
```